src/main/java/com/revature/ams/util/auth/AuthController.java

```diff
@@ -0,0 +1,36 @@
+package com.revature.ams.util.auth;
+
+import com.revature.ams.Member.Member;
+
+import javax.security.sasl.AuthenticationException;
+import java.util.Scanner;
+
+// TODO: REVIEW ME
+public class AuthController {
+
+    private final Scanner scanner;
+    private final AuthService authService;
+
+    public AuthController(Scanner scanner, AuthService authService) {
+        this.scanner = scanner;
+        this.authService = authService;
+    }
+
+    public Member login(Member memberLoggedIn){
+        try {
+            if(memberLoggedIn != null) throw new RuntimeException("Already logged in");
+            System.out.println("Please enter email: ");
+            String email = scanner.next();
+
+            System.out.println("Enter password: ");
+            String password = scanner.next();
+
+            return authService.login(email, password);
+
+        } catch (AuthenticationException | RuntimeException e) {
+            System.out.println(e.getMessage());;
```

Comment:
For now we're printing out the exception message, but in the future would we change this in any way? For example I know that in certain instances we might want to handle this exception by telling the user what they did wrong and actually display that message (in red) on the GUI.

Reply:
Logging will replace a lot of this material & eventual we will handle different situations via HTTP Responses to our users via postman or frontend once implemented